test/testcafe/spec/TerminalView_spec.js

```diff
@@ -140,8 +141,11 @@ async function setup(t) {
   test
     .requestHooks(mock)(testTitle, async t => {
       const terminalViewPage = await setup(t);
-      await t.expect(await terminalViewPage.goBackLinkExists()).notOk();
+
       await t.expect(await terminalViewPage.signoutLinkExists()).ok();
+      if(!process.env.OKTA_SIW_V3) {
```

Comment:
Env variables in NodeJS are always strings, so this probably won't work how we're expecting. Can you try copying over the approach I have in my other PR https://github.com/okta/okta-signin-widget/pull/2835 using the `userVariables` in TestCafe config? Would ensure we don't have to remember to set the env variable when running outside of a script, too.